packages/expo-notifications/src/InstallationIdProvider.web.ts

```diff
@@ -0,0 +1,21 @@
+import UUID from 'uuid-js';
+
+import { InstallationIdProvider } from './InstallationIdProvider';
```

Comment:
Put shared types in another module so we don't need to import NativeModulesProxy on web.